lock/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -366,13 +369,19 @@ public void onDatabaseAuthenticationRequest(DatabaseSignUpEvent event) {
         apiClient.setDefaultDatabaseConnection(configuration.getDefaultDatabaseConnection().getName());
 
         panelHolder.showProgress(true);
+
+
+        HashMap<String, Object> authParameters = options.getAuthenticationParameters();
+        if (event.extraFields() != null) {
+            authParameters.put(KEY_USER_METADATA, event.extraFields());
+        }
         if (event.loginAfterSignUp()) {
             apiClient.getProfileAfter(apiClient.signUp(event.getEmail(), event.getPassword(), event.getUsername()))
-                    .addParameters(options.getAuthenticationParameters())
+                    .addParameters(authParameters)
                     .start(authCallback);
         } else {
             apiClient.createUser(event.getEmail(), event.getPassword(), event.getUsername())
-                    .addParameters(options.getAuthenticationParameters())
+                    .addParameters(authParameters)
```

Comment:
This is actually wrong since the signup won't know how to do with them. The only parameters we need here are `'user_metadata`: {}`